### PR TITLE
Add integration tests for event handling cases in scheduler_perf

### PR DESCRIPTION
**Pull request by macsko**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Add small workloads to run as integration tests for event handling test cases. It should prevent bugs like described in #128666. 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: test/integration/scheduler_perf/event_handling/performance-config.yaml [hunk 1]
```diff
@@ -344,7 +355,12 @@
   - opcode: barrier
   - opcode: stopCollectingMetrics
   workloads:
-  - name: 100Nodes_500Pods
+  - name: 2Nodes_20Pods
+    labels: [performance, short]
```
Comment (dom4ha) [inline test/integration/scheduler_perf/event_handling/performance-config.yaml:359]: integration-test

**Review by dom4ha — COMMENTED** [review]
/lgtm

**Review by dom4ha — COMMENTED** [review]
/lgtm

**Review by sanposhiho — APPROVED** [review]
/lgtm
/approve

**Comment by sanposhiho** [comment]
So, the CI is failed because https://github.com/kubernetes/kubernetes/pull/128666 has to be merged before this PR?

/hold

**Comment by dom4ha** [comment]
/lgtm
reposted in case you're waiting for it

**Comment by macsko** [comment]
/unhold
https://github.com/kubernetes/kubernetes/pull/128666 is already merged

**Pull request merged**